modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/date/range/BaseDateRangeFacetPortletSharedSearchContributor.java

```diff
@@ -98,9 +98,12 @@ protected List<String> getSelectedRangeStrings(
 				selectedRangeStrings.add(rangesMap.get(selectedRange));
 			}
 			else {
-				selectedRangeStrings.add(
-					DateRangeFactoryUtil.getRangeString(
-						selectedRange, CalendarFactoryUtil.getCalendar()));
+				String rangeString = DateRangeFactoryUtil.getRangeString(
+					selectedRange, CalendarFactoryUtil.getCalendar());
+
+				if (Validator.isNotNull(rangeString)) {
```

Comment:
Since we are returning a string if null in `replaceAliases` method can we check for a blank here?

`!Validator.isBlank(rangeString)`